frontends/qt/Makefile

```diff
@@ -11,10 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+include ../../env.mk.inc
```

Comment:
a blank line above the include, please. license headers are usually separate with a blank line.

Reply:
done